sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -3264,18 +3264,13 @@ object GpuOverrides extends Logging {
           TypeSig.all))),
```

Comment:
There should be a PS note generated for struct and array noting that if they contain floating point types within they will not be supported.

Reply:
done.